torch/_utils.py

```diff
@@ -266,3 +266,13 @@ def _take_tensors(tensors, size_limit):
     for buf, _ in buf_dict.values():
         if len(buf) > 0:
             yield buf
+
+
+# annotation decorator to get annotations in a way that is compatible
+# with both Python 2 and 3
+def annotate(ret, **kwargs):
+    def dec(fun):
```

Comment:
nit: use @functools.wraps

Reply:
Do I need to use wraps when I don't actually replace fun?

Reply:
Oh you are right! Sorry about it. Ignore my comment above.